Services/Twilio/WorkflowConfiguration.php

```diff
@@ -21,7 +21,21 @@ public function toJSON() {
 	}
 
 	public static function parse($json) {
-		return json_decode($json);
```

Comment:
interesting. This seems like a backwards incompatible change. We should have a different function that parses into a workflowconfiguration to avoid a major version bump.